packages/webapp/src/routes/land/check-and-submit.js

```diff
@@ -56,7 +56,12 @@ const getContext = request => {
     getLegalAgreementDocumentType(request.yar.get(constants.redisKeys.LEGAL_AGREEMENT_DOCUMENT_TYPE)),
     legalAgreementFileNames: getLegalAgreementFileNamesForCheckandSubmit(applicationDetails.files),
     responsibleBodies: getResponsibleBodies(request.yar.get(constants.redisKeys.LEGAL_AGREEMENT_RESPONSIBLE_BODIES)),
-    landowners: getLandowners(request.yar.get(constants.redisKeys.LEGAL_AGREEMENT_LANDOWNER_CONSERVATION_CONVENANTS)),
+    landowners: request.yar.get(constants.redisKeys.ANY_OTHER_LANDOWNERS_CHECKED) === 'Yes'
+      ? getLandowners(request.yar.get(constants.redisKeys.LEGAL_AGREEMENT_LANDOWNER_CONSERVATION_CONVENANTS))
```

Comment:
i dont think this is right as we dont need to return on object if other landowner is yes. just need to return blank string '' and try to handle this in helper function as this lines are duplicated example 
const getLandowners = landOwners => {
  if (!landOwners) {
    // Handle the case where landOwners is null or undefined
    return '' // or return some default message
  }